### PR TITLE
Hark! No more need for `cogs.`!

### PR DESCRIPTION
**Pull request by GriffinG1**

### CODE REVIEW

**Comment by GriffinG1** [comment]
fixed @chenzw95

File: cogs/debugger.py [hunk 1]
```diff
@@ -328,30 +328,32 @@ def cleanup_code(self, content):
...
 
     @commands.command(pass_context=True)
     async def unload(self, ctx, *, msg):
         """Unload a module"""
         await ctx.message.delete()
         try:
-            if os.path.exists(msg.replace(".", "/") + ".py"):
-                self.bot.unload_extension(msg)
+            if os.path.exists("cogs/{}.py".format(msg)):
+                self.bot.unload_extension("cogs/{}.py".format(msg))
```
Comment (chenzw95) [inline cogs/debugger.py]: this should be `cogs.{}.py`.

Comment (GriffinG1) [reply cogs/debugger.py]: hm oh shit

**Comment by chenzw95** [comment]
For the record, in future I will be refusing PRs that contain such vague commit messages

**Comment by GriffinG1** [comment]
The original commit message was direct, and I'm pretty sure the other three devs would yell at you because of how vague their commit messages can be

**Pull request merged**